src/zui/ZUIText/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC } from 'react';
+import { Typography } from '@mui/material';
+
+type TextVariant =
+  | 'headingLg'
+  | 'headingMd'
+  | 'headingSm'
+  | 'bodyMdRegular'
+  | 'bodyMdBold'
+  | 'bodySmRegular'
+  | 'bodySmBold'
+  | 'linkMd'
+  | 'linkSm'
+  | 'labelMdMedium'
+  | 'labelMdRegular';
```

Comment:
This could be something along the lines of:

```tsx
const variants = {
  headingLg: {
    fontSize: 18, // or whatever the correct value is
    fontWeight: 'bold', // or whatever the correct value is
  },
  headingMd: {
    fontSize: 16, // or whatever the correct value is
    fontWeight: 'bold', // or whatever the correct value is
  },
} as const;
```